mal/core.py

```diff
@@ -277,6 +280,53 @@ def find(mal, regex, filtering='all', extra=False, user=None):
         anime_pprint(index + 1, item, extra=extra)
 
 
+def edit(mal, regex, changes):
+    """Select and change entry. Opens file with data to change if no
+    field was given."""
+    # find the correct entry to modify (handles animes not found)
+    entry = select_item(mal.find(regex, extra=True))
+
+    if not changes: # open file for user to choose changes manually
+        tmp_path = tempfile.gettempdir() + '/mal_tmp'
+        editor = os.environ.get('EDITOR', '/usr/bin/vi')
```

Comment:
I bet to set nano instead of vi... I prefer use vim instead nano, but vi is always present on any unix system? Like a raw arch linux. 🤔

Reply:
I'm not sure if it is included. vi was just the first to come to mind.
According to people on [this thread](https://unix.stackexchange.com/questions/194430/why-is-emacs-not-preinstalled) vi is part of the POSIX spec though.